mm/MemOps.c

```
@@ -25,245 +31,19 @@ int memcmp(const void* restrict s1, const void* restrict s2, const unsigned long
     return FastMemcmp(s1, s2, size);
 }
 
-void* FastMemset(void* restrict dest, int value, uint64_t size) {
+void* FastMemset(void* restrict dest, const int value, uint64_t size) {
```

Clang (14.0.6)

[warning] parameter 'size' is unused

(misc-unused-parameters)